backend/src/main/java/com/elshop/backend/common/FakerUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.elshop.backend.common;
+
+import com.elshop.backend.product.model.Product;
+import com.github.javafaker.Faker;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class FakerUtils {
+
+    private static final Faker faker = new Faker();
+
+    private static final AppUtils appUtils = new AppUtils();
+
+    private FakerUtils() {
+    }
+
+    public static Product generateFakeProduct(int imagesNr) {
+        List<String> images = generateImagesList(imagesNr);
+        String id = appUtils.generateUuid();
+        String title = faker.commerce().productName();
+        String slug = appUtils.generateSlug(title);
+
+        double price = Double.parseDouble(faker.commerce().price(1, 1000));
+        return new Product(id, slug, title, price, images, "Mobile");
+    }
+
+    public static List<Product> generateFakeProductsList(int productsNr) {
+
+        List<Product> list = Arrays.asList(new Product[productsNr]);
+
+        return list.stream()
+                .map(item -> generateFakeProduct(faker.number().numberBetween(1, 5)))
+                .toList();
```

Comment:
Ein raffinierter Weg um die Produkte zu erstellen. Da wir in diesem Fall jedoch die Liste eigentlich gar nicht brauchen - sondern nur einen Stream mit entsprechend vielen Einträgen, schlage ich vor:
```suggestion
        return IntStream.range(0, productsNr)
                .mapToObj(x -> generateFakeProduct(faker.number().numberBetween(1, 5)))
                .toList();
```